packages/ui-a11y-utils/src/ScreenReaderFocusRegion.ts

```diff
@@ -29,13 +29,13 @@ class ScreenReaderFocusRegion {
   private _parents: HTMLElement[] = []
   private _nodes: Element[] = []
   private _liveRegion: (Element | undefined)[]
-  private _contextElement?: Element | Node
+  private _contextElement: Element | Node | null
```

Comment:
Just a sidenote: I dont like that our code is using `undefined` and `null` to indicate non-existence of a value seemingly randomly. We should either stick to one or have some kind of rule when we use one.

Reply:
True, it is probably worth making a ticket for it to decide and solve it globally.